lib/Runtime/Library/ArgumentsObject.cpp

```diff
@@ -85,20 +85,17 @@ namespace Js
             return scriptContext->GetLibrary()->GetNull();
         }
 
-        Var args = walker->GetPermanentArguments();
-        if (args == NULL)
-        {
-            args = JavascriptOperators::LoadHeapArguments(
-                funcCaller,
-                paramCount - 1,
-                walker->GetJavascriptArgs(),
-                scriptContext->GetLibrary()->GetNull(),
-                scriptContext->GetLibrary()->GetNull(),
-                scriptContext,
-                /* formalsAreLetDecls */ false);
-
-            walker->SetPermanentArguments(args);
```

Comment:
Are these functions (Set/GetParmanentArgumetns) used anywhere else? You could remove them from the stack-walker if they are not.